shotover-proxy/tests/cassandra_int_tests/cluster/single_rack_v4.rs

```diff
@@ -63,7 +63,7 @@ async fn test_rewrite_system_peers_dummy_peers(connection: &CassandraConnection)
         ResultValue::Any,
         // Unfortunately token generation appears to be non-deterministic but we can at least assert that
         // there are 128 tokens per node
-        ResultValue::Set(std::iter::repeat(ResultValue::Any).take(3 * 128).collect()),
+        ResultValue::AnyList(std::iter::repeat(ResultValue::Any).take(3 * 128).collect()),
```

Comment:
I dont like that this makes tests against all drivers weaker when its only one driver that doesnt support sets properly.
Is there a reason our old workaround is no longer suitable?

Reply:
What was the old work around?

Reply:
https://github.com/shotover/shotover-proxy/blob/5a1558bc0ca35e63ec614569f2540db11aa33e2c/shotover-proxy/tests/cassandra_int_tests/collections.rs#L194